lib/gulpDepcheck.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const _ = require('lodash'),
+      dc = require('depcheck'),
```

Comment:
Ah, okay, I see… hmmm. What do you think about this one:

- The global one is `depcheck`, as it represents the module.
- The local one could be called `depcheckInstance`, as this now is the specific one you are going to deal with.

What do you think?